src/pages/umi/kit-adapters.md

```
@@ -0,0 +1,126 @@
+
+## Installation and Imports
+
+Install the adapters package:
+
+```
+npm i @metaplex-foundation/umi-kit-adapters
+```
+
+After installation, the conversion functions are available:
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)